### DISCUSSION TITLE
Error: Unexpected MODIFIER at 5, expected END

### DISCUSSION THREAD

**Discussion by smohadjer** [vercel/community#5039] [Bug Report]
### Page to Investigate

https://github.com/smohadjer/pokerrangliste

### Steps to Reproduce

My github project fails to deploy starting today and I can't figure out why. The error in Vercel log says:
````
Error: Unexpected MODIFIER at 5, expected END
````
I don't use any frameworks and my own npm build which uses esbuild to compile ts to js runs without any error locally. Any idea what this error means and what could be causing it?

**Comment by smohadjer** [comment]
I found the cause. It was due to using wildcard in my middleware matcher. I had it like this:
````
export const config = {
  matcher: '/api/*
}
````

And it seems the correct syntax is this:
````
export const config = {
  matcher: '/api/:path*'
}
````

**Reply by yotambarzilay** [reply]
Thank you for this! I just had the same issue, and couldn't figure it out.

A descriptive error message for invalid matcher values could have been really helpful

**Reply by smohadjer** [reply]
Yes, very cryptic message indeed! :)
